### PR TITLE
Added links and new styling

### PR DESCRIPTION
**Pull request by prashantiA**
- Changed fonts, colors, and stying
- Added links to my github and linkedin with icons
- Changed sizes to use rem instead of px
Screenshot 2020-05-28 at 1 30 14 PM - Display 2

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,76 @@
...
+        and can often be found listening to Shardcast or browsing the Seventeenth Shard. 
+      </p>
+      <img src="/images/honk_square.jpg" alt="Prashanti and Honk" class="circle centered"/>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
+    <hr>
+    <div id="proj-resume" class="full-height group">
+      <h1> Projects and Resume </h1>
```
Comment (lee12jin) [inline portfolio/src/main/webapp/index.html:68]: Is the empty spacing intentional?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,76 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Prashanti Anderson's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Cardo&family=Oswald:wght@300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/39869a2f70.js" crossorigin="anonymous"></script>
```
Comment (jasperlee-google) [inline portfolio/src/main/webapp/index.html:7]: Is this the CDN url? Is there a CDN url for fontawesome?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,76 @@
...
+  </nav>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+    <div id="top" class="full-height group">
+      <h1>Welcome to my personal portfolio!</h1>
+      <img src="/images/catgoose.png" alt="Prashanti Profile Picture" class="circle centered"/>
+      <h1 class="icon-links">
+        <a href="https://github.com/prashantiA"><i class="fab fa-github"></i></a>
```
Comment (jasperlee-google) [inline portfolio/src/main/webapp/index.html:24]: Is there a reason why this is an `<i>` element as opposed to a `<div>`?

Comment (prashantiA) [reply portfolio/src/main/webapp/index.html:24]: I'm not sure if the two are exchangable. The <i> element came from fontawesome's instructions for using their icons. Would I be able to switch it out for a div?

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (prashantiA) [reply portfolio/src/main/webapp/index.html:7]: Yeah this is the source of the icons assets.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (jasperlee-google) [reply portfolio/src/main/webapp/index.html:24]: SGTM, if this is directly from the docs then I would definitely keep it as is.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (jasperlee-google) [reply portfolio/src/main/webapp/index.html:7]: Alright wanted to check as sometimes there is a separate CDN url optimized for serving purposes with JS libs.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (prashantiA) [reply portfolio/src/main/webapp/index.html]: Removed the extra spaces!

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,76 @@
...
+	<a href="https://linkedin.com/in/prashanti-anderson"> <i class="fab fa-linkedin"></i></a>
```
[the remark this answers is not in the document]
Comment (prashantiA) [reply portfolio/src/main/webapp/index.html]: Removed the extra space!

**Pull request merged**